src/routes/docs/multix-walletconnect/+page.svelte

```diff
@@ -6,16 +6,16 @@
 			functionalities.
```

Comment:
The instructions in this file are the same in the landing page as far as I can tell, maybe we should avoid duplication and make a component of it?

Reply:
I somewhat anticipate the content on the home page to be changed, and then the component won’t make sense